### PR TITLE
New python file codebytes

### LINKED ISSUE #2819: [Edit] Python Add Codebytes
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

Please add Codebytes to the listed entries.
https://www.codecademy.com/resources/docs/python/files/close
https://www.codecademy.com/resources/docs/python/files/readline
https://www.codecademy.com/resources/docs/python/files/remove

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) when working on the PR for this issue.

Bear in mind the following two principles:
1) There should only be one Codebyte per entry listed as `## Codebyte Example` at the end of the entry.
2) The codebytes should be new examples, not copies of the prior examples.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Luke943**
### Description

Closes #2819 .

Codebytes have been added for three Python file methods: close, readline and remove.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/files/terms/close/close.md [hunk 1]
```diff
@@ -31,3 +31,17 @@ f = open("gullivers_travels.txt", "r", encoding='utf8')
 
 f.close()
 ```
+
+## Codebyte Example
+
+The example below opens a file called **example_file.txt** and then closes it using `.close()`. The `.closed` attribute is used to show whether the file is open or closed.
```
Comment (caupolicandiaz) [inline content/python/concepts/files/terms/close/close.md:37]:
```suggestion
The example below opens a file called `example_file.txt` and then closes it using `.close()`. The `.closed` attribute is used to show whether the file is open or closed.
```

File: content/python/concepts/files/terms/close/close.md [hunk 2]
```diff
@@ -31,3 +31,17 @@ f = open("gullivers_travels.txt", "r", encoding='utf8')
...
+
+```codebyte/python
+f = open("example_file.txt", "r+")
```
Comment (caupolicandiaz) [inline content/python/concepts/files/terms/close/close.md:40]:
```suggestion
filename = "example.txt"
f = open(filename, "w")

```

File: content/python/concepts/files/terms/readline/readline.md [hunk 3]
```diff
@@ -31,3 +31,16 @@ f = open("gullivers_travels.txt", "r", encoding='utf8')
 
 f.readline()
 ```
+
+## Codebyte Example
+
+The example below first creates a file called **sample_text.txt** and writes multiple lines of text to the file. It then calls `.readline()` twice to extract the two lines of content.
```
Comment (caupolicandiaz) [inline content/python/concepts/files/terms/readline/readline.md:37]:
```suggestion
The example below first creates a file called `sample_text.txt` and writes multiple lines of text to the file. It then calls `.readline()` twice to extract the two lines of content.
```

File: content/python/concepts/files/terms/remove/remove.md [hunk 4]
```diff
@@ -50,3 +50,19 @@ os.remove("my_file.txt")
 # Delete file in another directory
 os.remove("path/to/my_file.txt")
 ```
+
+## Codebyte Example
+
+The example below creates a file called **example.txt** and then deletes it using the `.remove()` method. The `.isfile()` method is used to show at which stage the file is found.
```
Comment (caupolicandiaz) [inline content/python/concepts/files/terms/remove/remove.md:56]:
```suggestion
The example below creates a file called `example.txt` and then deletes it using the `.remove()` method. The `.isfile()` method is used to show at which stage the file is found.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Luke943 I've added my comments. Please update the entries, and tag me when you've finished making changes. Thanks

**Comment by Luke943** [comment]
Thanks for the feedback @caupolicandiaz . I've made the changes you suggested to the files.

The Example sections of these files also use the markdown format **filename.txt** instead of `filename.txt`. Should they be changed as well?

**Review by SSwiniarski — APPROVED** [review]
@Luke943 Looks good.

**Pull request merged**